src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -37,6 +37,12 @@ public function __construct(Definition $definition, MarkingStoreInterface $marki
     {
         $this->definition = $definition;
         $this->markingStore = $markingStore ?: new MultipleStateMarkingStore();
+
+        $r = new \ReflectionMethod($this->markingStore, 'setMarking');
+        if (3 > \count($r->getParameters())) {
+            @trigger_error(sprintf('The MarkingStore "%s" should have a third arguments "array $context = []".', \get_class($this->markingStore)), E_USER_DEPRECATED);
```

Comment:
argument

Reply:
and we should talk about the method name here too

Reply:
this should be removed: see my comment above: `DebugClassLoader` will ensure this notice is triggered